app/helpers/window.js

```diff
@@ -65,18 +66,28 @@ export default function (name, options) {
         }
         return windowState;
     };
+	
+	function updateState() {
```

Comment:
This project uses convention `var updateState = function () {` for declaring functions. Convention is convention :)